common-lib/src/main/scala/com/gu/mediaservice/lib/elasticsearch/Mappings.scala

```diff
@@ -123,6 +123,57 @@ object Mappings {
       "filename" -> nonAnalyzedString
     )
 
+  val usageReference =
+    nonDynamicObj(
+      "type" -> nonAnalyzedString,
+      "uri"  -> nonAnalyzedString,
+      "name" -> sStemmerAnalysedString
```

Comment:
Back and forth over the code a few times and I don't really like any of the refactoring options very much so i'm going to leave it.